src/module.c

```diff
@@ -2544,7 +2549,7 @@ void VM_Yield(ValkeyModuleCtx *ctx, int flags, const char *busy_reply) {
         }
 
         /* decide when the next event should fire. */
-        ctx->next_yield_time = now + 1000000 / server.hz;
+        computeNextYieldTime(ctx);
```

Comment:
```suggestion
        ctx->next_yield_time = computeNextYieldTime();
```
It seems odd that this function both computes and sets it on a context, especially since it's not dependent on the context. Not a strong opinion though.

Reply:
good point. I prefer "function" over "method" too.